templates/cluster-template-machinepool-windows.yaml

```diff
@@ -270,4 +261,8 @@ spec:
       name: '{{ ds.meta_data["local_hostname"] }}'
   preKubeadmCommands:
   - powershell c:/create-external-network.ps1
-  - powershell c:/configure-ssh.ps1
+  users:
+  - groups: Administrators
+    name: capi
+    sshAuthorizedKeys:
+    - ${AZURE_SSH_PUBLIC_KEY:=""}
```

Comment:
is this is expecting plain text (not base64 encoded)? 😕

Reply:
yup. I had the same expression when I figured that out.  We push the ssh key through the Azure api but we could use the cloud-init for linux in the same way.

Reply:
@jsturtevant what happens when this is empty? For Linux there is a defaulting webhook that sets a random key but here that won't be the case... Do we need to update any e2e / conformance tests to have that var set?

Reply:
the sshkey will be blank:
``` 
 users:
  - groups: Administrators
    name: capi
    sshAuthorizedKeys:
    - ""
```

> Do we need to update any e2e / conformance tests to have that var set?

We don't currently collect the windows logs in e2e: https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/1286